styles/learn-x-in-y-minutes/catppuccin.user.css

```diff
@@ -0,0 +1,208 @@
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain("learnxinyminutes.com") {
+  @import url("https://unpkg.com/@catppuccin/highlightjs@0.2.2/css/catppuccin.variables.important.css");
+
+  html.light {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  html.dark {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  html:not(.dark),
+  html:not(.light) {
+    @media (prefers-color-scheme: light) {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+    @media (prefers-color-scheme: dark) {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[ @@lookup][ @rosewater];
+    @flamingo: @catppuccin[ @@lookup][ @flamingo];
+    @pink: @catppuccin[ @@lookup][ @pink];
+    @mauve: @catppuccin[ @@lookup][ @mauve];
+    @red: @catppuccin[ @@lookup][ @red];
+    @maroon: @catppuccin[ @@lookup][ @maroon];
+    @peach: @catppuccin[ @@lookup][ @peach];
+    @yellow: @catppuccin[ @@lookup][ @yellow];
+    @green: @catppuccin[ @@lookup][ @green];
+    @teal: @catppuccin[ @@lookup][ @teal];
+    @sky: @catppuccin[ @@lookup][ @sky];
+    @sapphire: @catppuccin[ @@lookup][ @sapphire];
+    @blue: @catppuccin[ @@lookup][ @blue];
+    @lavender: @catppuccin[ @@lookup][ @lavender];
+    @text: @catppuccin[ @@lookup][ @text];
+    @subtext1: @catppuccin[ @@lookup][ @subtext1];
+    @subtext0: @catppuccin[ @@lookup][ @subtext0];
+    @overlay2: @catppuccin[ @@lookup][ @overlay2];
+    @overlay1: @catppuccin[ @@lookup][ @overlay1];
+    @overlay0: @catppuccin[ @@lookup][ @overlay0];
+    @surface2: @catppuccin[ @@lookup][ @surface2];
+    @surface1: @catppuccin[ @@lookup][ @surface1];
+    @surface0: @catppuccin[ @@lookup][ @surface0];
+    @base: @catppuccin[ @@lookup][ @base];
+    @mantle: @catppuccin[ @@lookup][ @mantle];
+    @crust: @catppuccin[ @@lookup][ @crust];
+    @accent-color: @catppuccin[ @@lookup][ @@accent];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
```

Comment:
Pretty sure you need these to be `rgbify`-ed? See https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#highlightjs. Would be surprised if this works as is!

Reply:
Hmm this is strange, all of the code is now colourless

![image](https://github.com/user-attachments/assets/16c965f1-29fb-4716-98f6-86c9e2f750ca)

This wasn't an issue before! It had color (although the color was wrong)

I tried placing the variables at the end, marking them all with important, it didn't work. And also tried the `#rgbify` mixin.

Reply:
It will work if you use the mixin! Copy the variables from that linked section, and add the rgbify mixin to the bottom of the userstyle.

Reply:
Oh wait no your variables are correct, you just used the wrong import link at the top. You used Highlight.js, it should be Pygments.